README.md

```diff
@@ -210,7 +210,7 @@ import ops
 import scenario
 
 # ...
-ctx.run('start', scenario.State(unit_status=ops.ActiveStatus('foo')))
+ctx.run(ctx.start(), scenario.State(unit_status=ops.ActiveStatus('foo')))
```

Comment:
```suggestion
ctx.run(ctx.on.start(), scenario.State(unit_status=ops.ActiveStatus('foo')))
```

Reply:
Thanks! Opened #139 to catch these automatically in the future.